saliency/base.py

```diff
@@ -95,3 +98,83 @@ def GetMask(self, x_value, feed_dict={}):
     """
     feed_dict[self.x] = [x_value]
     return self.session.run(self.gradients_node, feed_dict=feed_dict)[0]
+
+
+OUTPUT_GRADIENTS = 'OUTPUT_GRADIENTS'
+CONVOLUTION_LAYER = 'CONVOLUTION_LAYER'
+CONVOLUTION_GRADIENTS = 'CONVOLUTION_GRADIENTUS'
+
+
+class CallModelSaliency(object):
```

Comment:
That makes sense, especially if we're thinking of adding utility functions for other frameworks. This would be a new folder in this repo with its own setup.py, correct?

Reply:
Split tf1 implementations into saliency.tf1.